modules/extensions-support/pom.xml

```diff
@@ -96,6 +96,10 @@
             <groupId>net.bytebuddy</groupId>
             <artifactId>byte-buddy</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+        </dependency>
```

Comment:
isn;t this transitive already? why need to ceclare it here?

Reply:
intellij fails to validate the module-info otherwise